portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -51,8 +60,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String name = getParameter(request, "reviewer-name", "");
```

Comment:
Discussed in chat, but be careful about using the empty string as a default, since there can be some weird edge cases.

Reply:
for sure, changing to null.